docs/devGuide/index.md

```diff
@@ -1 +1,3 @@
-<include src="devGuide.md" />
\ No newline at end of file
+<script>
+  window.location.href = "devGuide.html"
+</script>
```

Comment:
This seems to work.

But the issue is: The back button on the browse doesn't work on this page anymore. Every time you press the back button it brings you to the same page. This happens due to re-direction I feel.

@ang-zeyu Your POV on this.

Reply:
Hmm what browser is this? It seems to work fine for me on chrome, edge and firefox.
Did a little reading, `location.href =` should not add a new entry if it is not user initiated.

Otherwise you could try a more explicit version https://developer.mozilla.org/en-US/docs/Web/API/Location/replace

Reply:
I tried on Google Chrome too. But wasn't going back for some reason.

Reply:
I tried on Google Chrome again after closing the browser. Works okay!